ui/src/shared/components/TimeMachine/CSVExporter.tsx

```diff
@@ -39,19 +52,41 @@ class CSVExporter extends PureComponent<Props, State> {
     )
   }
 
-  private handleClick = async () => {
-    const {queries, templates, onNotify} = this.props
+  private handleClick = async (): Promise<void> => {
+    const {isFluxSource, onNotify} = this.props
 
     this.setState({buttonStatus: ComponentStatus.Loading})
 
     try {
-      await downloadTimeseriesCSV(queries, templates)
+      if (isFluxSource) {
+        await this.downloadFluxCSV()
```

Comment:
what if this fails?

Reply:
this and the one below will both hit the `catch` block and a notification will show